src/zalasr.adoc

```diff
@@ -20,7 +40,7 @@ If all accessed bytes lie within the same misaligned atomicity granule, the inst
 === Load Acquire
 
 Synopsis::
-The load-acquire instruction, atomically and subject to the ordering annotations specified in the instruction, loads a 2^width^-byte value from the address in _rs1_ into the register _rd_.
+The load-acquire instruction, atomically and subject to the ordering annotations specified in the instruction, loads a 2^width^-byte value from the address in _rs1_ into the register _rd_, sign-extending it.
```

Comment:
The load-acquire instruction atomically loads a 2^width^-byte value from the address in rs1 and places the sign-extended value into the register rd, subject to the ordering annotations specified in the instruction.